README.md

```diff
@@ -139,6 +140,8 @@ Further Information
 | elasticsearch_filesystem_io_stats_device_write_operations_count      | gauge      | 1           | Count of disk write operations                                                                      |
 | elasticsearch_filesystem_io_stats_device_read_size_kilobytes_sum     | gauge      | 1           | Total kilobytes read from disk                                                                      |
 | elasticsearch_filesystem_io_stats_device_write_size_kilobytes_sum    | gauge      | 1           | Total kilobytes written to disk                                                                     |
+| elasticsearch_ilm_status                                             | gauge      | 3           | Current status of ILM. Status can be `STOPPED`, `RUNNING`, `STOPPING`.                              |
```

Comment:
In this case, cardinality is the number of labels, not the number of possible values for the labels. ilm_status should be 1 and ilm_index_status should be 4.